src/main/scala/com/avast/marathon/plugin/vault/VaultPlugin.scala

```diff
@@ -12,47 +12,36 @@ import play.api.libs.json.{JsObject, _}
 
 import scala.util.{Failure, Success, Try}
 
-case class Configuration(address: String, token: String)
+case class Configuration(address: String, token: String, absolutePathRoot: Option[String], appRelativePathRoot: Option[String])
```

Comment:
I would prefer to have both configuration values as mandatory (so no Option).

Reply:
I would like defaults of both to be `/` (which is the most tolerant setting, but does not assume any particular structure in Vault)